doc/source/structures/misc/steeringmanager.rst

```diff
@@ -0,0 +1,233 @@
+
+.. attribute:: SteeringManager:TARGET
+
+    :type: :struct:`Direction`
+    :access: Get only
+
+    Returns direction that the is currently being targeted.  If steering is locked to a vector, this will return the calculated direction.  If steering is locked to "kill", this will return the vessel's last facing direction.
+
+.. method:: SteeringManager:RESETPIDS
+
+    :return: none
+
+    Returns direction that the is currently being targeted.  If steering is locked to a vector, this will return the calculated direction.  If steering is locked to "kill", this will return the vessel's last facing direction.
+
+.. attribute:: SteeringManager:SHOWFACINGVECTORS
+
+    :type: bool
+    :access: Get/Set
+
+    Setting this suffix to true will cause the steering manager to display graphical vectors (see :struct:`VecDraw`) representing the forward, top, and starboard of the facing direction, as well as the world x, y, and z axis orientation (centered on the vessel).  Setting to false will hide the vectors, as will disabling locked steering.
+
+.. attribute:: SteeringManager:SHOWANGULARVECTORS
+
+    :type: bool
+    :access: Get/Set
+
+    Setting this suffix to true will cause the steering manager to display graphical vectors (see :struct:`VecDraw`) representing the current and target angular velocities in the pitch, yaw, and roll directions.  Setting to false will hide the vectors, as will disabling locked steering.
+
+.. attribute:: SteeringManager:SHOWTHRUSTVECTORS
+
+    :type: bool
+    :access: Get/Set
+
+    Setting this suffix to true will cause the steering manager to display graphical vectors (see :struct:`VecDraw`) representing the thrust and torque for each active engine.  Setting to false will hide the vectors, as will disabling locked steering.
+
+.. attribute:: SteeringManager:SHOWRCSVECTORS
+
+    :type: bool
+    :access: Get/Set
+
+    Setting this suffix to true will cause the steering manager to display graphical vectors (see :struct:`VecDraw`) representing the thrust and torque for each active RCS block.  Setting to false will hide the vectors, as will disabling locked steering.
+
+.. attribute:: SteeringManager:SHOWSTEERINGSTATS
+
+    :type: bool
+    :access: Get/Set
+
+    Setting this suffix to true will cause the steering manager to clear the terminal screen and print steering data each update.
+
+.. attribute:: SteeringManager:WRITECSVFILES
+
+    :type: bool
+    :access: Get/Set
+
+    Setting this suffix to true will cause the steering manager log the data from all 6 PIDLoops calculating target angular velocity and target torque.  The files are stored in the `[KSP Root]\GameData\kOS\Plugins\PluginData\kOS` folder, with one file per loop and a new file created for each new manager instance (i.e. every launch, every revert, and every vessel load).  These files can grow quite large, and add up quickly, so it is recommended to only set this value to true for testing or debugging and not normal operation.
```

Comment:
Yes csv is documented.

One row per line.  Commas separate cells.  If you want to protect literal commas or literal EOLN marks, you do it by using quote marks around the value.  Two quote marks back to back is how you literalize a quote mark.